app/src/main/kotlin/cz/covid19cz/erouska/ui/login/LoginFragment.kt

```diff
@@ -90,12 +91,11 @@ class LoginFragment :
         enableUpInToolbar(true)
 
         login_statement.text = HtmlCompat.fromHtml(
-            getString(R.string.login_statement),
+            getString(R.string.login_statement, AppConfig.termsAndConditionsLink),
             HtmlCompat.FROM_HTML_MODE_LEGACY
         )
-        login_statement.setOnClickListener {
-            showWeb(AppConfig.termsAndConditionsLink)
-        }
+        login_statement.movementMethod = LinkMovementMethod()
```

Comment:
nemá to být náhodou  `login_statement.movementMethod = LinkMovementMethod.getInstance() ?`